test/AngularSpec.js

```diff
@@ -228,6 +228,18 @@ describe('angular', function() {
       }
     });
 
+    it('should handle Blob objects', function() {
+      if (typeof Blob !== 'undefined') {
+        var src = new Blob(['foo'], {type: 'bar'});
+        var dst = copy(src);
+
+        expect(dst).not.toBe(src);
+        expect(dst.size).toBe(3);
+        expect(dst.type).toBe('bar');
+        expect(isBlob(dst)).toBe(true);
```

Comment:
I'm fairly certain I got Blob with PhantomJS. Don't recall what actual user browsers gave me there. But it does not matter in the end. Should work like a charm now. :smiley_cat: